### PR TITLE
Changes to ubuntu install instructions to cover kilo

### PR DESCRIPTION
**Pull request by DaveLangridge**

### CODE REVIEW

File: docs/source/ubuntu-opens-install.rst [hunk 1]
```diff
@@ -53,17 +53,16 @@ Configuring the APT software sources
...
 
 Add the Calico PPA.
 
 ::
 
-    sudo apt-add-repository ppa:project-calico/icehouse
+    sudo apt-add-repository ppa:project-calico/<release>
+
 
+Where <release> is icehouse, juno or kilo
```
Comment (alexwlchan) [inline docs/source/ubuntu-opens-install.rst]: Note: to get \<release> to show up in a monospaced font, add backticks, like so: <code>\`\`\<release>``</code>. Also, this sentence needs a full stop.

**Comment by alexwlchan** [comment]
Do we need to add a new link to install instructions for Kilo at the top of the document, or are we assuming that everybody will have Icehouse or Juno already?

File: docs/source/ubuntu-opens-install.rst [hunk 2]
```diff
@@ -17,7 +17,7 @@ Ubuntu 14.04 Packaged Install Instructions
 
 These instructions will take you through a first-time install of Calico using
 the latest packages on a system running Ubuntu 14.04 with either OpenStack
-Icehouse or Juno. If you are upgrading an existing system, please see the
+Icehouse, Juno or kilo. If you are upgrading an existing system, please see the
```
Comment (alexwlchan) [inline docs/source/ubuntu-opens-install.rst]: “Kilo” should be capitalised.

**Comment by ed-harrison** [comment]
@DaveLangridge I've made a few more suggested changes.  One thing which I think would still be worth adding is a note on how to tell whether you've got the correct libvirt or not.  Could you add that and then pass this to @matthewdupre to merge (which we should do once the Kilo packages are actually ready)?

**Comment by DaveLangridge** [comment]
Matt, can you merge this when we're ready to release.

**Pull request merged**